jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/io/FrameFlusher.java

```diff
@@ -293,10 +295,10 @@ public String toString()
     private final int bufferSize;
     private final Generator generator;
     private final int maxGather;
-    private final Object lock = new Object();
+    private final Locker lock = new Locker();
     private final Deque<FrameEntry> queue = new ArrayDeque<>();
     private final Flusher flusher;
-    private final AtomicBoolean closed = new AtomicBoolean();
+    private boolean closed = false;
```

Comment:
Mixing Atomic state variables and locked blocks of code is pretty much an anti pattern.  Use one or the other and never both.